src/containers/signupForm/index.tsx

```diff
@@ -102,7 +116,7 @@ const SignupForm: React.FC<SignupFormProps> = ({ tokens }) => {
         />
       </Helmet>
       <ContentContainer>
-        {asyncRequestIsComplete(tokens) ? (
```

Comment:
What does this logical change do? this component is quite large 😅 
We should think about relying on ternaries less, using switches more..